code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -300,6 +300,77 @@
 	icon_state = "gold"
 	item_state = "gold_id"
 
+/obj/item/weapon/card/id/nt_disguise
+	name = "Nanotrasen undercover ID"
+	access = list(access_weapons, access_security, access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_eva)
+	registered_name = null
+
+	var/registered_user = null
+
+/obj/item/weapon/card/id/nt_disguise/attack_self(mob/user as mob)
+
+	var/list/gimmick_names = list(
```

Comment:
Personally, I'd put this in the object definition rather than inside the proc, but it's not a huge deal.